umbraco-cloud/set-up/project-settings/umbraco-cicd/KnownLimitationsAndConsiderations.md

```diff
@@ -18,7 +20,22 @@
 
 * Given the necessity to avoid changes in other environments, the lack of strict coordination among multiple teams or individuals working on the same project elevates the risk of conflicts.
 
-As we continue to prototype and gather insights from beta users, there are some known limitations and considerations to be aware of:
+**Usage of Private NuGet Feeds**
```

Comment:
as this limitation is quite big and has some steps to follow I suggest to add it as a sub-section under this "## Potential Limitations and Considerations" section. 

```suggestion
###Usage of Private NuGet Feeds
```